### PR TITLE
Support Jucer versions after 6.0.0

### PR DESCRIPTION
**Pull request by McMartin**
Projucer doesn't write the `jucerVersion` attribute since
https://github.com/juce-framework/JUCE/commit/91d9679f0, which was
released in JUCE 6.0.1. However, the code and behavior of Projucer has
changed significantly since that JUCE commit and `Reprojucer.cmake`
needs to know which version of Projucer it should mimic.

### CODE REVIEW

**Comment by McMartin** [comment]
@MartyLake please review, thanks!

File: ci/apply-Jucer2CMake-reprojucer-to-JUCE-jucers.cmake [hunk 1]
```diff
@@ -49,6 +49,8 @@ foreach(jucer_file IN LISTS jucer_files)
     "reprojucer"
     "${JUCE_ROOT}/${jucer_file}"
     "${CMAKE_CURRENT_LIST_DIR}/../cmake/Reprojucer.cmake"
+    "--jucer-version"
+    "${JUCE_VERSION}"
```
Comment (McMartin) [inline ci/apply-Jucer2CMake-reprojucer-to-JUCE-jucers.cmake:53]: Note to self: that's worth a copyright year bump

**Review by MartyLake — APPROVED** [review]
LGTM

File: ci/apply-Jucer2CMake-reprojucer-to-JUCE-jucers.cmake (hunk 1 above)
Comment (McMartin) [reply ci/apply-Jucer2CMake-reprojucer-to-JUCE-jucers.cmake:53]: Done in https://github.com/McMartin/FRUT/pull/715/commits/6a04e75edcfe687e583281abf15854870d31041c.

**Pull request merged**